docs: Add some notes about dialogs (#585)
oamg/leapp: docs/source/actor-rhel7-to-rhel8.md

By executing the above actor, all active kernel modules would be logged on output using log utilities inherited from the Actor class.

### Asking user questions

In rare cases the actor can't choose a proper scenario of execution and leaves the final decision of how to proceed
to the user. That's where interactive dialogs come into play.

Please mind that using dialogs should be considered as last resort, when the situation absolutely can't resolve itself

> automatically. The rule of a thumb is to make upgrade procedure require as less user input as possible. But if you

Reviewer: ```suggestion
automatically. The rule of a thumb is to make upgrade procedure require as little user input as possible. But if you
```

> feel that there is no way to write a proper safe rhel7->rhel8 convertion logic and you need human to make a decision -

Reviewer: ```suggestion
feel that there is no way to write a proper safe rhel7->rhel8 conversion logic and you need human to make a decision -
```

you can go with dialogs.

> Currently the following restrictions apply:

Reviewer: 'Currently' seems a bit redundant to me

- Only Yes/No questions can be asked. Effectively only leapp.dialogs.components.BooleanComponent can be used.

> - Dialogs can't be co-dependent. Any question asked should be independent of previous question's answer and should not

Reviewer: ```suggestion
- Dialogs can't be codependent. Any question asked should be independent of previous question's answer and should not
```
